resources/contribute-blog/_blog-template.md

```diff
@@ -0,0 +1,54 @@
+---
+category: 2025
+date: 2025-mm-dd
+seo:
+    title: SEO optimized title under 50chars
+    description: SEO optimized description for the blog post that can be displayed in search results.
+
+labels:
+    - General
+markdown:
+    editPage:
+        hide: true
+---
+# SEO optimized title under 50chars (Recommend keeping it same as the meta title above for consistency)
+
+
+Congrats on taking the first step toward drafting your blog post! Use this template as a starting point to organize your blog post.
+
+<!-- BREAK -->
+
+Refer to the [Contributing Guide](https://xrpl.org/resources/contribute-documentation) for detailed guidelines and examples.
```

Comment:
Refer to Contribute Documentation

Also, I don't see anything specific about blog posts on the page. We might consider adding something, if only to say "Use the template."

Whoops  - ah - okay, I see that there's a new Blog Post guide below. This should point to the Blog Post guide, which should then point to Contribute Documentation, IMO.